os_dep.c

```diff
@@ -96,6 +96,11 @@
 # include "private/darwin_stop_world.h"
 #endif
 
+#if defined(__CHERI_PURE_CAPABILITY__)
+# include <link.h>
+#endif
+
```

Comment:
Double blank line?

Reply:
changed in  [07b586b](https://github.com/capablevms/bdwgc/pull/19/commits/07b586b7ec011b520062977bd8e863a51061e711)